### PR TITLE
feat: added home page 

### PR DESCRIPTION
**Pull request by Tanjum786**
created e commerce landing page 
Screenshot from 2022-02-15 01-25-22
Screenshot from 2022-02-15 01-25-43

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
```
Comment (rahulcs754) [inline CSS/style.css]: I think you have to not redeclare if your declare in reset css in your lib.

**Review by rahulcs754 — APPROVED** [review]
looks all good but some suggestion if you define your class in component lib then don't redeclare
 here

**Review by kjais1720 — APPROVED** [review]
Looks good overall👍

File: CSS/style.css (hunk 1 above)
Comment (Tanjum786) [reply CSS/style.css]: ohh ,ok i will correct it thank you for pointing out

**Pull request merged**